src/test/scala/io/iohk/ethereum/pos/ElectionManagerImplSpec.scala

```diff
@@ -58,4 +54,44 @@ class ElectionManagerImplSpec extends FlatSpec with Matchers with PropertyChecks
     electionManager.verifyIsLeader(knownStakeholders(1), slotNumber = 3) shouldEqual false
     electionManager.verifyIsLeader(knownStakeholders(2), slotNumber = 3) shouldEqual false
   }
+
+  "ElectionManagerImpl" should "choose the correct miner, given a change in the stakeholder list" in new TestSetup {
```

Comment:
In order to tighten up the test, I would add assertions that specify exactly the set of stakeholders that are able to mine in each slot, e.g. 

    // First round stakeholders
    electionManager.verifyIsLeader(knownStakeholders(0), slotNumber = 1) shouldEqual true
    electionManager.verifyIsLeader(knownStakeholders(1), slotNumber = 1) shouldEqual false
    electionManager.verifyIsLeader(knownStakeholders(2), slotNumber = 1) shouldEqual false
    electionManager.verifyIsLeader(otherKnownStakeholders(0), slotNumber = 1) shouldEqual false
    electionManager.verifyIsLeader(otherKnownStakeholders(1), slotNumber = 1) shouldEqual false